technic/machines/network.lua

```diff
@@ -123,10 +132,28 @@ local function touch_node(tier, pos, timeout)
 		-- this should get built up during registration
 		node_timeout[tier] = {}
 	end
-	node_timeout[tier][poshash(pos)] = timeout or 2
+	node_timeout[tier][poshash(pos)] = timeout or 5
```

Comment:
This was updated to somewhat mitigate problems caused by throttling, last technic problems at pandorabox (machines showing/blinking no network often) were caused because:
* Timeout ABM frequency was set back to 1sec / 100% to make digtron better
* Throttling at max lag 1.5 or more causes technic run to have at least 3 sec intervals.
* Machines will time out between globalstep because ABM timeout is 2 seconds.

So changed this to 5 seconds for now to somewhat mitigate issues when throttling is active.

Reply:
Changed again to be dynamic based on current throttling...